homeassistant/components/group/registry.py

```diff
@@ -61,13 +78,8 @@ def __init__(self, hass: HomeAssistant) -> None:
         self.on_off_mapping: dict[str, str] = {STATE_ON: STATE_OFF}
         self.off_on_mapping: dict[str, str] = {STATE_OFF: STATE_ON}
         self.on_states_by_domain: dict[str, set[str]] = {}
-        self.exclude_domains: set[str] = set()
         self.state_group_mapping: dict[str, SingleStateType] = {}
-
-    @callback
-    def exclude_domain(self, domain: str) -> None:
```

Comment:
Should we keep this method to not make it a breaking change, and just make it a no op and log a warning that it's deprecated?

Reply:
There was already a breaking change as we changed the signature for the `registry` method (https://github.com/home-assistant/core/pull/116317). I do not expect this to break anything. The removal is mentioned in the blog post though.